Rakefile

```diff
@@ -0,0 +1,67 @@
+# Rakefile
+# Copyright (C) 2018 Mike JS Choi
+# Copyright (C) 2018 VLC authors and VideoLAN
+# $Id$
+#
+# Authors: Mike JS. Choi <mkchoi212 # icloud.com>
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston MA 02110-1301, USA.
+#
+#
+#
+# ------------------------------------------------------------- Constants ------
+
+SDK_SIM = "iphonesimulator11.3"
+SDK_SIM_DEST = "'platform=iOS Simulator,name=iPhone 7,OS=11.3'"
+
+SCHEME_IOS = "MobileVLCKit"
```

Comment:
right now the testing and build scheme are called the same which is very confusing. Can we rename it to also include the name test?

Reply:
👍

Reply:
So I added `MobileVLCKit-iOSTests` as one of the testable schemes in `Manage Schemes` in Xcode but because of `project.xcworkspace` in `.gitignore`, I can't commit the change 🤔